src/platforms/php/common/performance.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Performance
+description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
+---
+
+: `traces_sampler`
+
+If you want to sample specific Transaction, use the `traces_sampler` option:
+
+```php
+Sentry\init([
+    'dsn' => '___PUBLIC_DSN___',
+    'traces_sampler' => static function (Sentry\Tracing\SamplingContext $context): float {
+        if (false !== strpos($context->getTransactionContext()->getName(), 'health')) {
+            // Discard transactions that have 'health' in their name
+            return 0.0;
+        }
+        // Sample rate for all other transactions
+        return 1.0;
+    },
+]);
+```
+
+it can also be defined as a callable which receives a `Sentry\Tracing\SamplingContext $context` and should return a `float` (for the sample rate):
+
+```php
+'traces_sampler' => array('App\Helpers\General\TracesSampler', 'sample')
+```
+
+### Instrument http calls with GuzzleHttp
+
+You need to add a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html#handlers) to your Guzzle Client. We provide one within the SDK which can be used like this:
+
+```php
+use Sentry\Tracing\GuzzleTracingMiddleware;
+...
+
+$stack = HandlerStack::create();
+
+// This is the important line
+$stack->push(GuzzleTracingMiddleware::trace());
+// ----
+
+$client = new Client([
+    'base_uri' => 'http://httpbin.org',
+    'timeout'  => 2.0,
+    'handler' => $stack
+]);
+$client->request('GET', '/get');
+```
+
+After adding this handler, you will receive spans for every request done with this client.
+
+### Manual Instrumentation
+
+To manually instrument certain regions of your code, you can create a transaction to capture them.
+
+```php
+$transactionContext = new TransactionContext();
+$transactionContext->setName('External Call');
+$transactionContext->setOp('http.caller');
+$transaction = \Sentry\startTransaction($transactionContext);
+
+$spanContext = new SpanContext();
+$spanContext->setOp('functionX');
+$span1 = $transaction->startChild($spanContext);
+
+// Calling functionX
+$this->functionX();
+$span1->finish();
+
+$transaction->finish();
+```
+
+### Retrieving a Transaction
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `SentrySdk::getCurrentHub()->getTransaction()`. This function will return a `Transaction` in case there is a running Transaction on the scope, otherwise it returns `null`.
```

Comment:
```suggestion
If you want to attach Spans to an already ongoing Transaction, you can use `SentrySdk::getCurrentHub()->getTransaction()`. This function will return a `Transaction` if there is a running Transaction on the scope, otherwise it returns `null`.
```